session-replay/Sources/DatadogSessionReplay/DatadogCoreIntegration/RUMDependency.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// Defines dependency between Session Replay and RUM modules.
```

Comment:
Just for clarity so people can know for sure that SR is referring to Session Replay:
```suggestion
/// Defines dependency between Session Replay (SR) and RUM modules.
```